### PR TITLE
Capif demo next steps

### PR DESCRIPTION
**Pull request by sxarism**

### CODE REVIEW

File: python_aef/certs/server-req.pem [hunk 1]
```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE REQUEST-----
```
Comment (raxonros) [inline python_aef/certs/server-req.pem]: don't push server cert to GitHub

File: python_aef/certs/server-key.pem [hunk 2]
```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (raxonros) [inline python_aef/certs/server-key.pem]: don't push private key to GitHub

File: python_aef/certs/server-cert.pem [hunk 3]
```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```
Comment (raxonros) [inline python_aef/certs/server-cert.pem]: don't push cert to GitHub

File: python_aef/certs/myCA.pem [hunk 4]
```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```
Comment (raxonros) [inline python_aef/certs/myCA.pem]: don't push ca to GitHub

File: python_aef/certs/server-req.pem (hunk 1 above)
Comment (sxarism) [reply python_aef/certs/server-req.pem]: done

File: python_aef/certs/server-key.pem (hunk 2 above)
Comment (sxarism) [reply python_aef/certs/server-key.pem]: done

File: python_aef/certs/server-cert.pem (hunk 3 above)
Comment (sxarism) [reply python_aef/certs/server-cert.pem]: done

File: python_aef/certs/myCA.pem (hunk 4 above)
Comment (sxarism) [reply python_aef/certs/myCA.pem]: done

**Pull request merged**